benchmarks/src/bench/find_many.rs

```diff
@@ -21,40 +19,43 @@ pub struct Options {
     pub uri: String,
 }
 
+#[async_trait::async_trait]
 impl Benchmark for FindManyBenchmark {
     type Options = Options;
 
-    fn setup(options: Self::Options) -> Result<Self> {
-        let client = Client::with_uri_str(&options.uri)?;
+    async fn setup(options: Self::Options) -> Result<Self> {
+        let client = Client::with_uri_str(&options.uri).await?;
         let db = client.database(&DATABASE_NAME);
-        db.drop()?;
+        db.drop(None).await?;
 
-        let mut file = File::open(options.path)?;
+        let num_iter = options.num_iter;
 
-        let json: Value = serde_json::from_reader(&mut file)?;
-        let doc = match json.into() {
+        let mut file = spawn_blocking_and_await!(File::open(options.path))?;
```

Comment:
One interesting part of the Rust async ecosystem is that only network I/O is truly async; the filesystem APIs offered by tokio and async-std are actually just wrappers around their APIs for spawning blocking tasks. Since it would require more work to abstract over each libraries filesystem APIs, I opted to just write a macro that called the `spawn_blocking` function from whichever runtime is enabled and use around the standard library filesystem API. With one exception in the `json_multi_export` tests (see the comment below), this didn't require any functional changes to the benchmarks. We can always come back later and do the work to properly use the runtime-specific filesystem APIs, but I didn't think it was a high enough priority to be worth taking time away from the other parts of this project to do that right now.